reconcile/github_org.py

```diff
@@ -83,23 +84,36 @@
 QONTRACT_INTEGRATION = 'github'
 
 
-def get_config(desired_org_name=None):
+def get_config(default=False):
     gqlapi = gql.get_api()
     orgs = gqlapi.query(ORGS_QUERY)['orgs']
     settings = queries.get_app_interface_settings()
     secret_reader = SecretReader(settings=settings)
     config = {'github': {}}
+    default_found = False
     for org in orgs:
         org_name = org['name']
-        if desired_org_name and org_name != desired_org_name:
+        is_default_org = org.get('default')
+        if default and not is_default_org:
             continue
+        if default_found:
+            raise KeyError('multiple default github org configs found')
```

Comment:
Print the found defaults in the exception, it'll make troubleshooting much easier.

Reply:
OTOH It'd be even easier if you added all the found defaults to a list and raised at the end if `len(all_defaults) > 1`.

Reply:
done in https://github.com/app-sre/qontract-reconcile/pull/2160/commits/fac34618dd6d40ab51fa31ea2d9e8338eb0fbed9

please take another look